internal/controller/api_to_config.go

```diff
@@ -523,6 +525,20 @@ func joinRawConfigs(raw []namedRawConfig) string {
 	return res.String()
 }
 
+func alwaysBlockToFRR(cidrs []net.IPNet) []frr.IncomingFilter {
+	res := make([]frr.IncomingFilter, 0, len(cidrs))
+	for _, c := range cidrs {
+		family := ipfamily.ForCIDR(&c) // #nosec G601
```

Comment:
why use nosec?

Reply:
the warning is about the pointer being referenced in a loop. The risk, I guess, is to carry around the pointed object which changes while looping. In this case, there's no risk as ForCIDR is a pure function that does not depend on &c but just returns its family. We could've added a local copy to make gosec happy but imo that would've been a waste of resources.